cirq/aqt/aqt_sampler_test.py

```diff
@@ -0,0 +1,142 @@
+import mock
+import numpy as np
+import pytest
+import sympy
+
+from cirq import LineQubit, X, Y, Z, XX, Circuit, study
+from cirq.aqt import AQTSampler, AQTSamplerSim
+from cirq.aqt.aqt_device import get_aqt_device
+
+
+class engine_return:
```

Comment:
PascalCase for class names

Reply:
This is something we can configure in the pylint config if you wish.

Reply:
Nevermind,  Pylint might not be that clever:  http://pylint.pycqa.org/en/latest/user_guide/options.html